### DISCUSSION TITLE
Position AlertService

### DISCUSSION THREAD

**Discussion by expdts** [taiga-family/taiga-ui#8170] [Q&A] [answered]
Подскажите, как для [AlertService ](https://taiga-ui.dev/services/alert-service) я могу установить свои значения для `TUI_ALERT_POSITION`, в документации к сожалению не нашел примера.

**Comment by waterplea** [comment, marked as answer]
Это просто токен, куда можно запровайдить `margin` для алертов. По дефолту там такое значение:
```ts
inject(TUI_IS_MOBILE) ? '1rem 1rem 0 auto' : '2rem 3rem 0 auto',
```
Если `auto` окажется справа, к примеру, то алерты будут появляться по левому краю. Если `auto` будет сверху, то алерты прилипнут к низу.
